tango/step.py

```diff
@@ -279,9 +279,6 @@ def _run_with_work_dir(self, cache: "StepCache", **kwargs) -> T:
 
         logger.info("Starting run for step %s of type %s", self.name, self.__class__.__name__)
 
-        if self.DETERMINISTIC:
-            random.seed(784507111)
```

Comment:
This potentially causes the unique ID for non-deterministic steps to be deterministic... which is bad.